spec/values/calculation/acos.hrx

```diff
@@ -0,0 +1,170 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `acos()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> less_than_negative_one/input.scss
+a {b: acos(-2)}
+
+<===> less_than_negative_one/output.css
+a {
+  b: calc(NaN * 1deg);
+}
+
+<===>
+================================================================================
+<===> negative_one/input.scss
+a {b: acos(-1)}
+
+<===> negative_one/output.css
+a {
+  b: 180deg;
+}
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: acos(0)}
+
+<===> zero/output.css
+a {
+  b: 90deg;
+}
+
+
+<===>
+================================================================================
+<===> one/input.scss
+a {b: acos(1)}
+
+<===> one/output.css
+a {
+  b: 0deg;
+}
+
+<===>
+================================================================================
+<===> greater_than_one/input.scss
+a {b: acos(2)}
+
+<===> greater_than_one/output.css
+a {
+  b: calc(NaN * 1deg);
+}
+
+<===>
+================================================================================
+<===> operation/simplification/input.scss
+a {
+  b: acos(3px - 1px + var(--c));
+}
+
+<===> operation/simplification/output.css
+a {
+  b: acos(2px + var(--c));
+}
```

Comment:
Since this is the only test for simplification behavior, and the point is just to test that simplification is happening at all, I think just calling it `simplification` makes the most sense.